tests/foreman/ui/test_computeprofiles.py

```diff
@@ -42,13 +42,19 @@ def test_positive_end_to_end(session, module_loc, module_org):
     ).create()
     with session:
         session.computeprofile.create({'name': name})
-        assert session.computeprofile.search(name)[0]['Name'] == name
+
+        assert entities.ComputeProfile().search(query={'search': 'name={0}'.format(name)}), \
+            'Compute profile {0} expected to exist, but is not included in the search'\
+            'results'.format(name)
```

Comment:
Please avoid using backslashes, they are discouraged by PEP8.
```python
        assert entities.ComputeProfile().search(query={'search': 'name={0}'.format(name)}), (
            'Compute profile {0} expected to exist, but is not included in the search '
            'results'.format(name)
        )
```
(Note additional space at the end of first line of string - breaking string into two lines doesn't add this space automatically, no matter if you use parentheses or backslash)

Reply:
sorry, getting used to new IDE

Reply:
Just to let you know, current version still has a lot of backslashes. I'm pointing this out because I'm not sure if that's intentional or not.